src/python/string/punctuation.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function punctuation () {
+  // Original by: Clari (https://github.com/Clari-7744)
+  //   example 1: punctuation()
+  //   returns 1: !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
+
+  return "!\"#$%&'()*+,-./:;<=>?@[\]^_`{|}~"
```

Comment:
if the characters should include a backslash, then the part `[\]` should probably be `[\\]`, otherwise it's escaping the `]`.

Reply:
right thanks, missed that one